src/udp.h

```diff
@@ -38,7 +38,9 @@ struct enftun_udp
 };
 
 int
-enftun_udp_connect_addr(struct enftun_udp* udp, struct sockaddr* addr);
+enftun_udp_connect_addr(struct enftun_udp* udp,
+                        struct sockaddr* addr,
+                        int mark);
```

Comment:
Put the `mark` parameter before the `addr` to match the `enftun_tcp_connect_*` calls.  Either order would be fine; let's just be consistent.